tests/postgres/trows.nim

```diff
@@ -219,7 +220,7 @@ suite "Row CRUD":
 
     let
       personRow = get dbConn.getRow(sql"""SELECT name, pet, id FROM "Person" WHERE id = $1""", person.id)
-      petRow = get dbConn.getRow(sql"""SELECT species, favToy, id FROM "Pet" WHERE id = $1""", pet.id)
+      petRow = get dbConn.getRow(sql"""SELECT species, "favToy", id FROM "Pet" WHERE id = $1""", pet.id)
```

Comment:
An example for suddenly required quotation marks, necessary only on the FK field "favToy", not on the others

Reply:
What if you put the rest in quotations for consistency? Does it break the test?

This is so weird 🤔 Possibly, this has to do with mixing cases. If the column name uses mixed case, it must be quoted. But that's just a wild guess.

Reply:
Changing the line to this:  `sql"""SELECT "species", "favToy", "id" FROM "Pet" WHERE id = $1"""`
Still works perfectly fine.

An observation I made though, is that this:
`sql"""SELECT "sPecies", "favToy", "id" FROM "Pet" WHERE id = $1"""`
Will cause the test to fail while this:
`sql"""SELECT sPecies, "favToy", "id" FROM "Pet" WHERE id = $1"""`
works perfectly fine.

I'm getting the impression that not only are suddenly the quotation marks sometimes mandatory, they cause the column names to suddenly be case-sensitive.